src/mysql.erl

```diff
@@ -130,6 +129,7 @@
                    {host, inet:socket_address() | inet:hostname()} | {port, integer()} |
                    {user, iodata()} | {password, iodata()} |
                    {database, iodata()} |
+                   {connect_mode, synchronous | asynchronous | lazy} |
```

Comment:
The option is missing in the edoc above.

Reply:
Added in latest commit, explanation has become quite verbose but should cover everything a user may need to know.